packages/cli/src/commands/setup/i18n/templates/i18n.js.template

```diff
@@ -34,10 +34,8 @@ export default HomePage
 */
 
 i18n
+  .use(LanguageDetector) // learn more: https://github.com/i18next/i18next-browser-languageDetector
   .use(initReactI18next)
-  // detect user language
```

Comment:
Why did you have to do this change?

Reply:
@Tobbe , I think it is ok to leave in this PR.

I looked at the respective docs and the`'react-i18next'` (`initReactI18next`) [setup example](https://react.i18next.com/latest/using-with-hooks#configure-i18next) uses the order that @simoncrypta changed it to.

See also: https://github.com/i18next/i18next-browser-languageDetector#adding-it